addons/stock_account/tests/test_stock_valuation_layer_revaluation.py

```diff
@@ -125,6 +125,78 @@ def test_stock_valuation_layer_revaluation_avco_rounding(self):
         self.assertEqual(len(credit_lines), 1)
         self.assertEqual(credit_lines[0].account_id.id, self.stock_valuation_account.id)
 
+    def test_stock_valuation_layer_revaluation_avco_rounding_2_digits(self):
+        self.product1.categ_id.property_cost_method = 'average'
+        context = {
+            'default_product_id': self.product1.id,
+            'default_company_id': self.env.company.id,
+            'default_added_value': 0.0
+        }
+        # Quantity of product1 is zero, raise
+        with self.assertRaises(UserError):
+            Form(self.env['stock.valuation.layer.revaluation'].with_context(context)).save()
+
+        self.env['decimal.precision'].search([
+            ('name', '=', 'Product Price'),
+        ]).digits = 2
+        self.product1.write({'standard_price': 0})
+
+        # First Move
+        self.product1.write({'standard_price': 0.022})
+        self._make_in_move(self.product1, 10000)
+
+        self.assertEqual(self.product1.standard_price, 0.02)
+        self.assertEqual(self.product1.quantity_svl, 10000)
+
+        layers = self.env['stock.valuation.layer'].search([('product_id', '=', self.product1.id)], order="create_date desc, id desc")
+        self.assertEqual(layers[0].value, 200)
+
+        # Second Move
+        self.product1.write({'standard_price': 0.053})
+
+        self.assertEqual(self.product1.standard_price, 0.05)
+        self.assertEqual(self.product1.quantity_svl, 10000)
+
+        layers = self.env['stock.valuation.layer'].search([('product_id', '=', self.product1.id)], order="create_date desc, id desc")
+        self.assertEqual(layers[0].value, 300)
+        self.assertEqual(layers[1].value, 200)
+
+    def test_stock_valuation_layer_revaluation_avco_rounding_3_digits(self):
```

Comment:
You can remove it
One test is enough (and this test already works without the fix)